modules/create-resource-init-bundle.adoc

```diff
@@ -16,16 +17,36 @@ ifeval::["{context}" == "init-bundle-other"]
 :kube:
 endif::[]
 
-ifeval::["{context}" == "init-bundle-cloud-other"]
+ifeval::["{context}" == "init-bundle-cloud-other-apply"]
 :kube:
 endif::[]
 
+ifeval::["{context}" == "init-bundle-cloud-other-apply"]
+:cloud:
+endif::[]
+
+ifeval::["{context}" == "init-bundle-cloud-ocp-apply"]
+:openshift:
+endif::[]
+
+ifeval::["{context}" == "init-bundle-cloud-ocp-apply"]
+:cloud:
+endif::[]
 
-:_module-type: PROCEDURE
+:content-type: PROCEDURE
 [id="create-resource-init-bundle_{context}"]
 = Creating resources by using the init bundle
 
+//Do not show for ACSCS
+ifndef::cloud[]
 Before you install secured clusters, you must use the init bundle to create the required resources on the cluster that will allow the services on the secured clusters to communicate with Central.
```

Comment:
```suggestion
Before you install secured clusters, you must use the init bundle to create the required resources on the cluster that allows the services on the secured clusters to communicate with Central.
```

Reply:
resources is plural so "allow" is needed